float64 force
bool isClose
---
